changelog/unreleased/changes/2689--load-all-plugins-by-default.md

```diff
@@ -0,0 +1 @@
+If no "plugins" configuration key is defined in the configuration file or launch parameter, VAST will by default load all plugins now. This can be circumvented by specifying a "plugins" configuration key with no values.
```

Comment:
Three small requests for this:
- Can you reflow this text to 80 columns to align with the other Markdown files in the repository?
- Please refer to configuration options with their full name and use Markdown code blocks, i.e., `vast.plugins` rather than "plugins."
- Use active voice and present tense in documentation and changelog entry.

Reply:
I improved the changelog formatting in https://github.com/tenzir/vast/pull/2689/commits/ed9a3a0a8ccdae0c13e7103a9b32e77f2da056d6

Reply:
It's still passive voice. I recommend the following:

> VAST now loads all plugins by default. To revert to the old behavior, explicitly set the `vast.plugins` option to have no value.

Reply:
I applied your suggestion in https://github.com/tenzir/vast/pull/2689/commits/b8f4424f45dc1ea20698e8de9add1bb3f95f4761